include/openmc/mesh.h

```diff
@@ -965,8 +966,9 @@ class LibMesh : public UnstructuredMesh {
                          //!< during intialization
   vector<unique_ptr<libMesh::PointLocatorBase>>
     pl_; //!< per-thread point locators
-  unique_ptr<libMesh::EquationSystems>
-    equation_systems_; //!< pointer to the equation systems of the mesh
+  unique_ptr<libMesh::EquationSystems> equation_systems_ =
+    nullptr; //!< pointer to the libMesh EquationSystems
```

Comment:
No need to explicitly have `= nullptr` since the default constructor will do that already.